tests/helper/performance_helper.js

```diff
@@ -0,0 +1,14 @@
+const Helper = codecept_helper;
+
+class PerformanceHelper extends Helper {
+  async getPageTimeToLoad(page) {
+    await page.waitForLoadState('load');
+    const performanceTimingJson = await page.evaluate(() => JSON.stringify(window.performance.timing));
```

Comment:
Wouldn't it be easier to just 
```suggestion
    const performanceTiming = await page.evaluate(() => window.performance.timing));
```
instead of using two variables and parsing json back and forth? Or is there some javascript magic?

Reply:
javascript magic, fails otherwise.